integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -25,7 +26,7 @@ def __init__(
         mongo_connection_string: Secret = Secret.from_env_var("MONGO_CONNECTION_STRING"),  # noqa: B008
         database_name: str,
         collection_name: str,
-        recreate_collection: bool = False,
```

Comment:
Why remove this?

Reply:
Mentioned in the PR description:

>  Note: recreate_collection was removed because it doesn't make too much sense with how MongoDB Atlas works. Collections need to be created manually from the web UI to work with an embedding retriever, and right now this document store has no other retrieval methods. When APIs are introduced to create these collections and indices from code, or when we introduce alternative retrieval methods, we can evaluate whether to add this feature back.

Reply:
Ok, cool.